src/components/code-editor/code-editor.tsx

```diff
@@ -121,11 +121,13 @@ export class CodeEditor {
 
     private createEditor() {
         const options = this.getOptions();
+        const editorElement = this.host.shadowRoot.querySelector('.editor');
 
-        const editor = CodeMirror(
-            this.host.shadowRoot.querySelector('.editor'),
-            options
-        );
+        editorElement.childNodes.forEach((child) => {
+            child.remove();
```

Comment:
Feels to me like we could do this cleanup in the `disconnectedCallback`, unless there's a particular reason not to?

Reply:
Yeah probably. Do you want me to move it?

Reply:
Fastest reply ever 😂

Yeah, try moving it. Otherwise, some day when we come back to this code, we'll assume there was some reason it couldn't be in the `disconnectedCallback`, and scratch our heads over it for no reason 😆

Reply:
Fixed it and squashed it since the change was quite small :slightly_smiling_face: